### DISCUSSION TITLE
Display help as a tooltip

### DISCUSSION THREAD

**Discussion by mrleblanc101** [formkit/formkit#1567] [Ideas]
Hi,
It would be nice if we could customize the display of the "help" section. 
I'd like to display the help as a tooltip, both globally or per-component.

<img width="296" alt="Screenshot 2024-12-05 at 5 51 12 PM" src="https://github.com/user-attachments/assets/7e80b006-a6d6-4aff-a0c2-725e05b95ee3">

This is quite a common pattern, I'm surprised I couldn't find any similar issue.
Maybe this is possible or with a plugin and I just haven't found it.

**Comment by Riiiad** [comment]
Its possible to do this via plugin. Here is an example: https://formkit.link/9756b6de8376de698f6f7c04f135292c
This could even work for a Modal if you have more Information.

**Reply by mrleblanc101** [reply]
Thanks, I'll look into using Floating-vue instead of a custom tooltip as I already have the package installed and custom tooltip themes in my application that I would like to reuse
